lightfield_render.py

```diff
@@ -1387,24 +1387,24 @@ def cancel(self, context):
 		# CLEAN-UP FILES
 		# +++++++++++++++++++++++++++++++++++++++++++
 		# if the view files shall not be kept OR (still was rendered AND no filename was specfied) OR the file keeping is forced OR the incomplete render job was discarded
-		if ((self.render_settings.addon_settings.render_output == '1' or (not ((self.render_settings.job.animation == False and not self.render_settings.job.file_use_temp) or self.animation == True))) and self.render_settings.job.file_force_keep == False) or self.discard_lockfile == True:
+		# if ((self.render_settings.addon_settings.render_output == '1' or (not ((self.render_settings.job.animation == False and not self.render_settings.job.file_use_temp) or self.animation == True))) and self.render_settings.job.file_force_keep == False) or self.discard_lockfile == True:
 
-			LookingGlassAddonLogger.info("Cleaning up the disk files.")
+			# LookingGlassAddonLogger.info("Cleaning up the disk files.")
 
-			# if it was an animation
-			if self.render_settings.job.animation:
+			# # if it was an animation
+			# if self.render_settings.job.animation:
 
-				# for all frames of the animation
-				for frame in range(self.render_settings.job.scene.frame_start, self.render_settings.job.scene.frame_end + self.render_settings.frame_step):
+			# 	# for all frames of the animation
+			# 	for frame in range(self.render_settings.job.scene.frame_start, self.render_settings.job.scene.frame_end + self.render_settings.frame_step):
 
-					# delete views
-					self.render_settings.job.delete_files(frame)
+			# 		# delete views
+			# 		self.render_settings.job.delete_files(frame)
 
-			# if it was a still image
-			elif not self.render_settings.job.animation:
+			# # if it was a still image
+			# elif not self.render_settings.job.animation:
 
-				#  delete its views
-				self.render_settings.job.delete_files()
+			# 	#  delete its views
+			# 	self.render_settings.job.delete_files()

```

Comment:
This must not be commented out. If the user wants to keep the view files in addition to the quilt, they can use the already present UI option.

Reply:
Sorry, a co-worker made these changes at the beginning of the project and I failed to catch them before I made the request. I'll reverse these changes and re-test the plug in.

Reply:
Yes, both of these changes are undone in a later commit.

We originally planned to hack together a one time solution so changed the name and hard removed functionality we didn't want. 

I have checked these sections of the code directly and they are the same as they were before. Unfortunately, there are a few instances like this, for example, I change all of the 'LookingGlassAddonLogger.info's to print statements because I didn't understand how to get them in the terminal - This has also been undone.